src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -802,6 +802,57 @@ static int _xccdf_item_recursive_gather_selected_rules(struct xccdf_policy *poli
 	return ret;
 }
 
+static char *_comment_multiline_text(char *text)
```

Comment:
Please split this function into smaller functions - it is somehow mind-bending to guess the intent of the code.

Unfortunately, we don't have easy unit-test support, so please include a comment indicating what the functions is supposed to do - it may not be obvious outside of the context of this PR.

Reply:
This function would be a perfect candidate for an unit test!

Reply:
Yes. and I think that you can rename it to `trim_trailing_whitespace` or `rstrip`, as  multiline comments are out of scope of this function.